packages/visual-editor/src/hooks/useEditorSubscriber.ts

```diff
@@ -171,16 +175,19 @@ export function useEditorSubscriber() {
                 ...changedNode.data.unboundValues,
               });
             }
+
+            // Update the tree when all necessary data is fetched and ready for rendering.
```

Comment:
Should we move this back? I'm not sure if there is still an intention in repeating this code here.